src/server/common/helpers/start-server.js

```diff
@@ -1,23 +1,16 @@
 import { config } from '~/src/config'
 
 import { createServer } from '~/src/server'
-import { createLogger } from '~/src/server/common/helpers/logging/logger'
 
 async function startServer() {
-  try {
-    const server = await createServer()
-    await server.start()
+  const server = await createServer()
+  await server.start()
 
-    server.logger.info('Server started successfully')
-    server.logger.info(
-      `Access your frontend on http://localhost:${config.get('port')}`
-    )
-    return server
-  } catch (error) {
-    const logger = createLogger()
-    logger.info('Server failed to start :(')
-    logger.error(error)
-  }
+  server.logger.info('Server started successfully')
+  server.logger.info(
+    `Access your frontend on http://localhost:${config.get('port')}`
+  )
+  return server
```

Comment:
was the try/catch redundant or removed as part of debugging the dev/restart issue?

Reply:
This work has been moved to the catch in the added to the `startServer.catch()` call - this for the moment seems to of fixed the issue

Reply:
When we move to `ESM` I'll replace this with a high level `await`, same same as the templates